Source/WarpX.cpp

```diff
@@ -323,6 +324,17 @@ WarpX::ReadParameters ()
         pp.query("restart", restart_chkfile);
     }
 
+    {
```

Comment:
Could you remove this block of code, and instead use the one in the original position? (see comment below)